src/fragments/lib/analytics/flutter/getting-started/12_amplifyConfig.mdx

```diff
@@ -1 +1,9 @@
-Upon completion, `amplifyconfiguration.dart` should be updated to reference provisioned backend analytics resources. Note that these files should already be a part of your project if you followed the [Project setup walkthrough](/lib/project-setup/create-application).
+Upon completion, you can see the result as the following that is going to be providing you a link to reach your analytics data:
+
+```console
+✔ All resources are updated in the cloud
+
+Pinpoint URL to track events https://<YOUR-PROJECT-RELATED-URL>/analytics/overview
+```
+
+In addition, `amplifyconfiguration.dart` should be updated to reference provisioned backend analytics resources. Note that these files should already be a part of your project if you followed the [Project setup walkthrough](/lib/project-setup/create-application).
```

Comment:
Nit: "`amplifyconfiguration.dart` **will** be updated to reference provisioned backend...."